docs/content/preview/manage/backup-restore/instant-db-cloning.md

```diff
@@ -0,0 +1,292 @@
+
+    ```sql
+    production_db=# SELECT (EXTRACT (EPOCH FROM CURRENT_TIMESTAMP)*1000000)::decimal(38,0);
+    ```
+
+    ```output
+         numeric
+     ------------------
+      1723243720285350
+      (1 row)
+    ```
+
+1. Drop the table `t1` simulating a user error dropping the table accidentally.
+
+    ```sql
+    production_db=# DROP TABLE t1;
+    ```
+
+    ```output
+    DROP TABLE
+    ```
+
+1. Meanwhile, as table `t2` is still accepting reads/writes, you can insert 2 more rows as follows:
+
+    ```sh
+    INSERT INTO t2 (key,c1) SELECT i,md5(random()::text) FROM generate_series(6,7) AS i;
+    ```
+
+1. Now, if you try to query table `t1`, notice that the table is dropped and there is no way you can query it.
+
+    ```sql
+    production_db=# SELECT * FROM t1 ORDER BY k;
+    ```
+
+    ```output
+    ERROR:  relation "t1" does not exist
+    LINE 1: SELECT * FROM t1 ORDER BY k;
+    ```
+
+1. To get back lost data, create a database `clone_db` using `production_db` as the template and using the timestamp generated in step 4.
+
+    ```sql
+    production_db=# CREATE DATABASE clone_db TEMPLATE production_DB AS OF 1723243720285350;
+    ```
+
+    ```sql
+    \c clone_db
+    ```
+
+    ```output
+    You are now connected to database "clone_db" as user "yugabyte".
+    ```
+
+    ```sql
+    clone_db=# SELECT * FROM t1 ORDER BY k;
+    ```
+
+    ```output
+     k | v
+    ---+---
+     1 | 1
+     2 | 0
+     3 | 1
+     4 | 0
+     5 | 1
+    (5 rows)
+    ```
+
+    `clone_db` contains all the data from `production_db` at the specified timestamp, which means you can read table `t1` that was dropped accidentally. You can copy the lost data from table `t1` back to `production_db` by exporting the data from the clone and importing it into `production_db`. Alternatively, you can switch the workload to `clone_db`. You now have two isolated databases that can serve reads and writes independently.
+
+## Best practices
+
+Although creating a clone database is quick and initially doesn't take up much added disk space as no data is copied, it's important to understand that a clone does create an independent set of logical tablets. This effectively doubles the number of tablets, although the clone database tablets share the same data files as the original database. Keep in mind the following impacts:
+
+- Higher CPU usage due to the additional tablets
+- Increased memory consumption from the extra tablets
+- Increased space usage only if compaction is performed on clone or original database. This is because both original and post-compaction data files are kept in disk, that is, say compaction is performed on the source database which generates new compacted files. The old data files are kept to serve reads for the clone database.
```

Comment:
@yamen-haddad 
when the clone is deleted I guess all the old uncompacted files are cleaned up?

Reply:
Depending on where we performed the compaction (the clone vs the original database). As a general rule, a data file is deleted when it is no longer needed by either the original or the clone database.
What about adding this to the sentence:
`Whenever the clone or original database is deleted, the cluster only cleans the unused data files.`